main.cpp

```diff
@@ -107,15 +125,20 @@ int main()
 	std::stringstream failed_combo_names("ARG! DAMMIT! DARN! NOO!");
 	const std::vector<std::string> failed_combos = initialize_combolist(failed_combo_names);
 
+	std::vector<std::string> wordlist;
+	initialize_wordlist("words.txt", wordlist);
+
 	// Word to be guessed
-	const std::string_view word = "Oxygen";
+	const size_t random_word_index = randomRange(0, wordlist.size());
+	const std::string random_word = wordlist[random_word_index];
+	const std::string_view word = random_word;
 	const size_t len_of_word = word.size();
 
 	// used fill a letter in hangman
 	const std::string word_copy(word);
 
 	// # of tries before game over
-	const size_t tries = randomRange(len_of_word, len_of_word + 10);
+	const size_t tries = randomRange(len_of_word, (len_of_word + 20));
 
```

Comment:
Why is `tries` randomised?

Reply:
While testing out the game with this particular word list it took me a couple more tries to guess some words so I'd thought I'd increase it